scripts/perf/delete_horicromtal/delete_horicromtal.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+
+source scripts/perf/helper.inc.sh
+
+mkdir -p mnt
+
+# Read the service account credentials from vault:
+read_service_account_from_vault
+
+for instance in runner101 runner102 reader101
```

Comment:
Would it make sense to parameterize this as a list so it's potentially easier to maintain when we start to vary (perhaps dynamically!) the number of runners?

Reply:
I agree, yes, but... would you mind if we made that a follow up (vote with 👍) vs a blocker on this PR (vote with 👎)?